src/setup.js

```diff
@@ -27,9 +27,33 @@ function setupMeasure(dataLayer) {
    * The DataLayerHelper to use with this application.
    * @private @const {!DataLayerHelper}
    */
-  const helper = new DataLayerHelper(dataLayer);
+  const helper = new DataLayerHelper(dataLayer, {'processNow': false});
 
-   // TODO: Use the variables declared above
+  helper.registerProcessor('config', configProcessors);
+  helper.process();
+
+
+  // TODO: configProcessors jsdoc and better processor/storage init in PR 52
+  function configProcessors(eventProcessor, eventOptions,
+                            storageInterface, storageOptions) {
+    let processor = new eventProcessor(eventOptions);
+    let storage = new storageInterface(storageOptions);
+
+    /**
+     * Process an event object by sending it to the registered event processor
+     * along with interfaces to the storage and data layer helper.
+     *
+     * @param {string} name The name of the event to process.
+     * @param {!Object<string, *>=} args The arguments to
+     *     process with the event.
+     */
+    function processEvent(name, args = undefined) {
```

Comment:
Could you perhaps pass an empty object as the default parameter for args? This will help me not have to type check this argument!

Reply:
The default is an empty object (See line 51). It's written this way because of https://docs.google.com/document/d/1XIhihp4elYAa9cFqJL9vg3f7u4kbsoZqIQa5ytBqYg0/edit#heading=h.x95stkrs2rc2

Reply:
Today I learned! Thanks Alex :)